client/plots/hierCluster.config.js

```diff
@@ -161,6 +161,40 @@ export function makeChartBtnMenu(holder, chartsInstance) {
 					return tw
 				})
 			)
+
+			if (tws.length == 1) {
```

Comment:
may do away with this makeChartBtnMenu() of hierCluster, and implement the Gene Expression chart button logic in a consistent way as metabolicIntensity as in mass/charts.js